src/main/resources/db/migration/V2__Create_emplyee.sql

```diff
@@ -11,4 +11,5 @@ CREATE TABLE employee (
 
 INSERT INTO employee( first_name, last_name, email, start_date, created, updated)
 VALUES
+    ( 'Testi', 'Tekijä', 'testi@tekija.fi', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
```

Comment:
Why a duplicate? We should also probable require email to be unique or something to make sure the generated id isn't the only identifying field.

Reply:
Yes, definitely should have some a unique field! Currently `email` is nullable. And as far as this would make it possible to have identical entries if the `email` is set to null. Would it be better to make email non-nullable or just let it be like this?

Reply:
And the second row was just for the testing te delete. Definitely not good that it is identical to the first row, I will change it.

Reply:
Yeah email shouldn't be nullable. At least I think it's safe to assume every employee has an email. You can also make the column unique. Another thing would be to write a custom validator for the controller to make sure controller only accepts valid emails, but we'll do that in a different PrR.